pages/content/amp-dev/community/roadmap.html

```diff
@@ -99,7 +99,7 @@ <h2 class="ap-t-roadmap-filter-headline">{{ _('Select working groups') }}</h2>
           </script>
         </amp-animation>
         <button class="ap-t-roadmap-navigation-list-item"
-            on="tap:{{ quarter|slug }}.scrollTo(duration=250)">
+            on="tap:{% if loop.first %}status-updates{% else %}{{ quarter|slug }}{% endif %}.scrollTo(duration=500)">
```

Comment:
This could be simplified to `{{ 'status-updates' if loop.first else quarter|slug }}`